default.cfg.example

```diff
@@ -133,6 +133,9 @@ hideCoins = True
 #Plugins allow extending Bot functionality with extra features.
 #plugins = AccountStats
 
+#[ACCOUNTSTATS]
+#ReportInterval = 1800
```

Comment:
Need to update documentation as well.
https://github.com/BitBotFactory/poloniexlendingbot/blob/master/docs/configuration.rst#accountstats-plugin